### PR TITLE
docs: maven central alias readme fix

### PR DESCRIPTION
**Pull request by rsvoboda**
Maven central alias readme fix

### CODE REVIEW

**Comment by rsvoboda** [comment]
@maxandersen I don't know what went wrong on GH actions, but the tests passed on my local macOS.
And I believe that my changes don't have potential to break the tests :)

**Comment by maxandersen** [comment]
They break the semantics. It is effectively always enabling mavencentral. 

So if you can give me a reproducer of what fails for you that would be great.

**Comment by rsvoboda** [comment]
Reproducer: `JBANG_REPO=$HOME/Downloads/jb-repo-custom jbang --verbose ~/Downloads/quarkusrest.java`

```java
//usr/bin/env jbang --native "$0" "$@" ; exit $?

//REPOS mavencentral,local=file:///Users/rsvoboda/Downloads/maven-repository

//DEPS io.quarkus:quarkus-bom:1.11.0.Final-redhat-00002@pom
//DEPS io.quarkus:quarkus-resteasy
//DEPS io.quarkus:quarkus-smallrye-openapi

import javax.ws.rs.GET;
import javax.ws.rs.Path;

import static java.lang.System.*;
import javax.ws.rs.GET;
import javax.ws.rs.Path;
import io.quarkus.runtime.Quarkus;

@Path("/hello")
public class quarkusrest {

    public static void main(String... args) {
        Quarkus.run(args);
    }

    @GET
    public String sayHello() {
        return "hello from Quarkus with jbang.dev";
    }
}
```

or just upstream one
```java
//usr/bin/env jbang --native "$0" "$@" ; exit $?

//REPOS mavencentral

//DEPS io.quarkus:quarkus-bom:1.11.0.Final@pom
//DEPS io.quarkus:quarkus-resteasy
//DEPS io.quarkus:quarkus-smallrye-openapi

import javax.ws.rs.GET;
import javax.ws.rs.Path;

import static java.lang.System.*;
import javax.ws.rs.GET;
import javax.ws.rs.Path;
import io.quarkus.runtime.Quarkus;

@Path("/hello")
public class quarkusrest {

    public static void main(String... args) {
        Quarkus.run(args);
    }

    @GET
    public String sayHello() {
        return "hello from Quarkus with jbang.dev";
    }
}
```

**Comment by rsvoboda** [comment]
@maxandersen the reproducer works well with version 0.66.0, didn't work well with v0.65.1., reducing PR just to docs fix in few minutes

File: readme.adoc [hunk 1]
```diff
@@ -621,7 +621,7 @@ Following example enables use of Maven Central and add a custom `acme` repositor
 
 [source,java]
 ----
-//REPOS mavenCentral,acme=https://maven.acme.local/maven
+//REPOS mavencentral,acme=https://maven.acme.local/maven
```
Comment (maxandersen) [inline readme.adoc:624]: the lookup should be case insensitive ?

Comment (maxandersen) [reply readme.adoc:624]: is it not working for you?

**Review by maxandersen — CHANGES_REQUESTED** [review]

File: readme.adoc (hunk 1 above)
Comment (rsvoboda) [reply readme.adoc:624]: Works, but it's inconsistency with the table just above the this example where `mavencentral` is listed

**Comment by maxandersen** [comment]
@all-contributors please add @rsvoboda  for docs

**Comment by maxandersen** [comment]
@all-contributors please add @rsvoboda for docs

**Pull request merged**